packages/site/src/layout/AnimatedPresenceDisclosure.tsx

```diff
@@ -14,27 +14,40 @@ function AnimatedPresenceDisclosure({
   to,
 }: AnimatedPresenceDisclosureProps) {
   const [isOpen, setIsOpen] = useState(false);
+  const [isHovered, setIsHovered] = useState(false);
 
   const handleButtonClick = (e: React.MouseEvent) => {
     e.preventDefault();
     setIsOpen(!isOpen);
   };
 
+  const handleHover = (e: React.MouseEvent) => {
+    e.preventDefault();
+    setIsHovered(!isHovered);
+  };
+
   return (
-    <>
+    <div>
       <Link
         to={to ?? "/"}
         href="https://www.google.com"
         tabIndex={0}
         onFocus={e => (e.currentTarget.style.boxShadow = "0 0 3px #0073e6")}
         onBlur={e => (e.currentTarget.style.boxShadow = "none")}
+        onMouseEnter={handleHover}
+        onMouseLeave={handleHover}
         style={{
           display: "flex",
           alignItems: "center",
           justifyContent: "space-between",
-          backgroundColor: "var(--color-surface--background)",
+          backgroundColor: isHovered
+            ? "var(--color-surface)"
```

Comment:
- Should have a transparent (or `--color-surface--background`) in rest state, and `--color-surface--background--hover` when hovered
![image](https://github.com/user-attachments/assets/da41bd1a-c45a-445a-aadb-95e79f1182e5)

- the `surface` (white in light mode) is for the current page state

Reply:
Oh okay! I am a bit confused because the Figma shows the backgrounds as white when hovered on? Is this what it is supposed to be? 
![Screenshot 2024-12-11 at 12 02 42 PM](https://github.com/user-attachments/assets/576cf207-a3d5-4d20-9ad8-dc07da4cae9e)